langserver/util/util.go

```diff
@@ -15,6 +16,19 @@ func trimFilePrefix(s string) string {
 	return strings.TrimPrefix(s, "file://")
 }
 
+// ConcatCommentGroups joins the resultant comment text from two CommentGroups with a newline.
+func ConcatCommentGroups(a, b *ast.CommentGroup) string {
```

Comment:
can this just be an unexported helper function in the langserver package rather?